src/test/scala/uk/gov/hmrc/test/ui/pages/common/DashboardPage.scala

```diff
@@ -62,6 +62,10 @@ object DashboardPage extends BasePage {
     findChildByClassName(mrnCell, "govuk-link")
   }
 
+  def refreshPage(): Unit = {
```

Comment:
you can remove this

Reply:
This is required to refresh the page after we do amend